contracts/contracts/VerificationGateway.sol

```diff
@@ -340,6 +340,30 @@ contract VerificationGateway
         return IWallet(blsWallet);
     }
 
+    /**
+     * Utility for measuring the gas used by an operation, suitable for use in
+     * the operation's required gas parameter.
+     *
+     * This has two important differences over standard estimateGas methods:
+     *   1. It does not include gas for calldata, which has already been paid
+     *   2. It works for wallets which don't yet exist
+     */
+    function measureOperationGas(
+        uint256[BLS_KEY_LEN] memory publicKey,
+        IWallet.Operation calldata op
+    ) external returns (uint) {
+        // Don't allow this to actually be executed on chain. Static calls only.
+        require(msg.sender == address(0));
+
+        IWallet wallet = getOrCreateWallet(publicKey);
+
+        uint gasBefore = gasleft();
+        wallet.performOperation(op);
+        uint gasUsed = gasBefore - gasleft();
```

Comment:
The rest of our Solidity codebase specifies the precision of the uint, usually `uint256`. Is there a reason that is not done here?

Reply:
Good point. My subjective preference leaking in here. Will fix.